packages/core-sdk/resolvers/token-resolvers/resolvers/bancor/src/query/utils/getRate.ts

```diff
@@ -0,0 +1,67 @@
+import { Ethereum_Connection, Ethereum_Query } from "../w3";
+import { Big } from "as-big";
+import { BigInt } from "@web3api/wasm-as";
+
+// Bancor V2.1
+// https://github.com/bancorprotocol/contracts-solidity/blob/master/contracts/converter/types/standard-pool/StandardPoolConverter.sol#L925
+// function removeLiquidityReturn(uint256 amount, IReserveToken[] memory reserves) external view returns (uint256[] memory)
+// returns [rate after fee]
+
+export function getRateV2_1(
```

Comment:
I see you updated this, Is it because the old way you implemented wasn't providing the right answer?